General queries/System Guard Security Level Drop.txt

```diff
@@ -0,0 +1,24 @@
+// System Guard Security Level Drop
+// This query looks at the value of SystemGuardSecurityLevel in the DeviceBootAttestationInfo action type records in MiscEvents
+// when it sees a drop in the value of SystemGuardSecurityLevel of the latest record verus the previous record
+// You can convert this into a custom detection by selecting the "Create Detection Rule" in Advanced Hunting
+// For more questions on this query, feel free to ping @DepletionMode or @FlyingBlueMonki on twitter
+MiscEvents
+| where ActionType == "DeviceBootAttestationInfo"
+| partition by MachineId 
```

Comment:
I got an error when trying to run it on a large tenant:
`'partition' operator: exceeded amount of maximum partitions allowed (64)`

Reply:
I will look into this one and see what's going on.